lib/global.js

```diff
@@ -0,0 +1,27 @@
+/**
```

Comment:
I don't think it's worth it to make an entire function just to call it once. You can just set module.exports in the different branches.

Reply:
I can’t say I know the performance implications but I wanted to try and reduce side effects but it sounds like the benefits may not be worth it.

Reply:
I don't understand what you mean. There're no side effects here...

Reply:
I was thinking mutating the exports object as a side effect vs working with return values from the function but as of today I can see I'm probably over-engineering here. Will do as you suggest 🙂